program/write-a-program-to-check-anagram-string/WriteAProgramToCheckAnagramString.swift

```diff
@@ -0,0 +1,20 @@
+func sortedString(str: String) -> String {
+    let str1 = str.sorted();
+    let str2 = String(str1)
+    return str2
+}
+
+func AreAnagram(str1: String, str2: String) -> Void{
+    let str1 = sortedString(str: str1)
+    let str2 = sortedString(str: str2)
+    if str1 == str2 {
```

Comment:
can you please convert the string to lowercase before comparing , so that this will work for case sensitive strings as well .